src/screens/Drafting/Draft.tsx

```diff
@@ -31,36 +31,48 @@ const DraftScreen = ({ navigation }: any): ReactElement => {
 
   interface FormValues {
     date: string;
+    dateIsEnabled: boolean;
     companyName: string;
+    companyIsEnabled: boolean;
     address: string;
+    addressIsEnabled: boolean;
     contactPerson: string;
+    contactIsEnabled: boolean;
     phone: string;
+    phoneIsEnabled: boolean;
     jobPosition: string;
+    jobPositionIsEnabled: boolean;
     languageRequirement: string;
+    languageRequirementIsEnabled: boolean;
     workingHours: string;
+    workingHoursIsEnabled: boolean;
     workingDays: string;
+    workingDaysIsEnabled: boolean;
     salary: string;
+    salaryIsEnabled: boolean;
     probationPeriod: string;
+    probationPeriodIsEnabled: boolean;
     employeeBenefit: string;
+    employeeBenefitIsEnabled: boolean;
     otherInfo: string;
+    otherInfoIsEnabled: boolean;
   }
   const { ...methods } = useForm();
 
   const onSubmit: SubmitHandler<FormValues> = async (data) => {
     const map = new Map<string, boolean>();
-    map.set('date', dateIsEnabled);
-    map.set('companyName', companyNameIsEnabled);
-    map.set('address', addressIsEnabled);
-    map.set('contactPerson', contactPersonIsEnabled);
-    map.set('phone', phoneIsEnabled);
-    map.set('jobPosition', jobPositionIsEnabled);
-    map.set('languageRequirement', languageReqIsEnabled);
-    map.set('workingHours', workingHoursIsEnabled);
-    map.set('workingDays', workingDaysIsEnabled);
-    map.set('salary', salaryIsEnabled);
-    map.set('probationPeriod', probationPeriodIsEnabled);
-    map.set('employeeBenefit', employeeBenefitIsEnabled);
-    map.set('otherInfo', otherInfoIsEnabled);
+    map.set('date', data.dateIsEnabled);
```

Comment:
can we make this more programmatic at all? making future changes to the job fields could be a pain